build.py

```diff
@@ -214,13 +295,41 @@ def _validate_android_args(args: argparse.Namespace):
                 )
 
         if args.cmake_generator != original_cmake_generator:
-            log.info(f"Setting CMake generator to '{args.cmake_generator}' for cross-compiling for Android.")
+            log.info(
+                f"Setting CMake generator to '{args.cmake_generator}' for cross-compiling for Android."
+            )
 
         # no C# on Android so automatically skip
         if not args.skip_csharp:
             args.skip_csharp = True
 
 
+def _validate_ios_args(args: argparse.Namespace):
+    if args.ios:
+        # TODO: cross compile for iOS on Linux?
+        if util.is_mac():
```

Comment:
Should it be an error if this is not a mac?

Reply:
I think it's probably simpler to just require a Mac host.